src-ts/tools/learn/welcome/WelcomePage.tsx

```diff
@@ -26,12 +26,16 @@ const WelcomePage: FC<WelcomePageProps> = (props: WelcomePageProps) => {
                             <div className={styles['hero-col']}>
                                 <h1>Welcome to Topcoder ACADEMY</h1>
                                 <p className={styles['hero-text']}>
-                                    Lorem ipsum dolor sit amet, consectetur adipiscing elit,
-                                    sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
-                                    Diam sit amet nisl suscipit. Ullamcorper sit amet risus nullam eget.
-                                    Etiam erat velit scelerisque in dictum non consectetur a.
-                                    Mauris pellentesque pulvinar pellentesque habitant morbi.
-                                    Tortor pretium viverra suspendisse potenti nullam ac.
+                                    Thank you for visiting the Topcoder Academy.
+                                    The Topcoder Academy will provide enhanced learning
+                                    opportunities to you, our Topcoder community.
+                                    These learning opportunities will take form as guided learning paths
+                                    where you will have the opportunity to learn new skills.
+                                    With these newly learned skills, you will be better prepared to be
+                                    successful in competing in challenges and Topcoder matches,
+                                    you will have greater opportunities for gig work placement and will 
+                                    improve your overall opportunity to earn on the Topcoder platform. 
+                                    Welcome to the Topcoder Academy, we look forward to learning with you.
```

Comment:
I feel like this sentence should end with an `!` instead of a `.` -- can you change it?